### PR TITLE
Add Bengali (BN)

### PR DESCRIPTION
**Pull request by MjAbuz**
Language - Internationalization Bengali (BN) for reference please check here https://github.com/MjAbuz/hugo-book/pull/2

### CODE REVIEW

**Comment by MjAbuz** [comment]
@nicksnyder looking forward to add Bengali (BN)

File: v2/example/active.bn.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+হ্যালোনাম = "Hello {{.Name}}"
```
Comment (MjAbuz) [inline v2/example/active.bn.toml:1]: We don't have caps and small letters , what to do about it?
Hello = হ্যালো
Name = নাম

**Review by MjAbuz — COMMENTED** [review]
@nicksnyder  what do you advise?

**Review by nicksnyder — COMMENTED** [review]
This is just an example file so I don’t see a reason to add an example for a new language. If something isn’t working well for a specific language then can you file an issue?

**Pull request closed without merging**